alerts/config.go

```diff
@@ -135,6 +187,11 @@ func (m *DurationMinutes) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+func (m *DurationMinutes) MarshalJSON() ([]byte, error) {
```

Comment:
Yes, that's fine. The docs call for 1-minute granularity, and well-behaved clients should be adhering to that. Those that send other values can live with the poorly-defined behavior :)

Or it can perhaps be filed under "be broad in what you accept, and strict in what you emit" if you prefer. :)

Reply:
Sure. I just wanted to make you aware in case some downstream operation was generated based upon that initial incoming data that was in fractional minutes. 

For example, if upon initial create operation (immediately after `UnmarshalJSON` of fractional minutes), we created a some other data (e.g. a cron job) using those fractional minutes (e.g. the cron job would fire off every second). 

If we don't do this (and won't in the future), then it isn't an issue.